### PR TITLE
Unify load_image() in lmnet

### PR DESCRIPTION
**Pull request by yd8534976**
As #420 , to reduce duplicated code and manage them easily. I unify all load_image()-like functions into a `load_image()` in `utils/image.py`. I only changed obvious RGB format cases.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by ruimashita** [comment]
@yd8534976 
Thanks 👍
In `pre_processor.py`,  we should implement class that can be used with` PRE_PROCESSOR` in config file, I think.
How about `load_image()` in `utils/image.py` ?

**Comment by yd8534976** [comment]
@ruimashita  Thanks! If so, I may have to add an extra  link in `output_template` to `utils/image.py`. I'm not sure which one is better. 🤔

**Comment by ruimashita** [comment]
> I may have to add an extra link in output_template to utils/image.py.

yes, we need to link in `output_template/utils/image.py` to `lmnet/lmnet/utils/image.py` for `output_template/python/run.py`

File: output_template/python/lmnet/utils/image.py [hunk 1]
```diff
@@ -0,0 +1 @@
+lmnet/lmnet/utils/image.py
```
Comment (ruimashita) [inline output_template/python/lmnet/utils/image.py]: I think this path should be relative path from  `output_template/python/lmnet/utils/image.py`, Maybe `../../../../lmnet/lmnet/utils/image.py` is correct path 🤔

Comment (yd8534976) [reply output_template/python/lmnet/utils/image.py]: @ruimashita Thanks! Fixed.

**Pull request merged**